solutions/project/hub-env/org-policies/exceptions/compute-disable-serial-port-access-except-hub-project.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
there will be a bug fix to remove these empty line 14 in the org-policies YAML files (it's causing kpt to re-arrange the top of the file when rendering)

might as well remove it (or fill it with something like `#########`) from these exceptions now?